### DISCUSSION TITLE
trouble getting Desktop audio tools to compile OSX

### DISCUSSION THREAD

**Discussion by SnazzyBlanket** [pschatzmann/arduino-audio-tools#1556] [Q&A] [answered]
examples/examples-desktop/README.md

I have been trying to build any of the desktop examples on a Mac M1. Cmake runs fine but when i try to run Make I get a giant amount of errors (for the 3 different included examples the errors are different. I was wondering if anyone has had any success building these in mac os? It would be a godsend to be able to test audio code on the computer. 

`
56 warnings and 2 errors generated.
make[2]: *** [CMakeFiles/generator.dir/generator.ino.o] Error 1
make[1]: *** [CMakeFiles/generator.dir/all] Error 2
make: *** [all] Error 2`

**Comment by pschatzmann** [comment, marked as answer]
I would appreciate if you would just paste only the 2 errors the next time!
As far as I can see you are still using an old version and if you upgrade, I would expect that it will work.

By the way this is also part of the general recommendations if [something is not working](https://github.com/pschatzmann/arduino-audio-tools/wiki/It's-not-working)!

**Comment by SnazzyBlanket** [comment]
sorry about that :( i was not sure which errors were important or which
ones were not.

an old version of the audio-tools repo? oh shoot yes its possible i tried
to build this in a folder I had not updated yet.

**Comment by pschatzmann** [comment]
Only the lines marked as errors are relevant, info and warnings can be ignored...
